public/app/features/canvas/runtime/scene.tsx

```diff
@@ -23,10 +23,16 @@ import {
 } from 'app/features/dimensions/utils';
 import { ElementState } from './element';
 import { RootElement } from './root';
+import { GroupState } from './group';
+
+export interface SelectionParams {
+  targets: Array<HTMLElement | SVGElement>;
+  group?: GroupState;
+}
 
 export class Scene {
   styles = getStyles(config.theme2);
-  readonly selection = new ReplaySubject<ElementState[]>(1);
+  readonly selection = new ReplaySubject<Array<ElementState | GroupState>>(1);
```

Comment:
do you need ElementState | GroupState?  they both extend ElementState, s any typing would be equivalent